QtImageViewer/README.md

```diff
@@ -32,19 +32,31 @@ A Qt install and mesa-common-dev Debian package are required to run this example
 
     `sudo apt install mesa-common-dev`
 
+6. Follow the build instructions.
+
+7. After building the QtImageViewer, move a qt.conf file into the build file for the QtImageViewer. 
```

Comment:
A description of the contents of the qt.file is added. How do I track an example of the file in a ./res directory? The problem I am having is how do get an example of the qt.conf file as qt image viewer is a paid application?